dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -574,6 +590,41 @@ def _correct_bias_kurtosis(match_count, biased_kurtosis):
                    (biased_kurtosis + 3) - 3 * (match_count - 1))
         return kurtosis
 
+    def _estimate_mode_from_histogram(self):
+        """
+        Estimates the mode of the current data using the
+        histogram. If there are multiple modes, returns
+        K of them (where K is defined in options given, but
+        5 by default)
+
+        :return: The estimated mode of the histogram
+        :rtype: list(float)
+        """
+        if not self._has_histogram:
+            return np.nan
+
+        bin_counts = self._stored_histogram['histogram']['bin_counts']
+        bin_edges = self._stored_histogram['histogram']['bin_edges']
+
+        # Get the K bin(s) with the highest frequency (one-pass):
+        cur_max = -1
+        highest_idxs = []
+        count = 0
+        for i in range(0, len(bin_counts)):
+            if bin_counts[i] > cur_max:
+                highest_idxs = [i]
+                cur_max = bin_counts[i]
+                count = 1
+            elif bin_counts[i] == cur_max and count < self._top_k_modes:
+                highest_idxs.append(i)
+                count += 1
+            if count == self._top_k_modes:
+                    break
```

Comment:
This break is tabbed too far forward

Reply:
good catch

Reply:
this needs to be a continue because if we had top_k_5 and we had the bin setup it would give the wrong result:
`bin_counts = [5] * 5 + [6] * 6` it would breakout after the 5th instance of 5 when the mode has a count of 6.